environments/backfillEvents.toml

```diff
@@ -2,8 +2,89 @@
     home     = "github.com/makerdao/vulcanizedb"
     name     = "transformerExporter"
     save     = true
-    transformerNames = []
+    transformerNames = [
+        "auction_file",
+        "deal",
+        "dent",
+        "deny",
+        "flip_file_cat",
+        "flip_kick",
+        "rely",
+        "tend",
+        "tick",
+        "yank",
+    ]
```

Comment:
Does adding this step need to happen in a separate PR? Just realized that the backfill events startup script expects an ending block number, but I guess we wouldn't have that until we've confirmed that we've deployed an execute process that is tracking the new address?

Reply:
I missed this comment earlier somehow. Yeah, you're right that we wouldn't have the ending block number until we merge this PR in. I _think_ that if this config change stays in, then the backfill process would try to start up, but wouldn't do anything because it would try to backfill from the GUSD deployment block to the currently configured ending block, which is 11108609 and before GUSD is deployed. Not sure if that would cause an error or not though 🤔